server/server.js

```diff
@@ -22,6 +24,82 @@ const upload = multer({ dest: 'server/assets/uploads/' })
 const app = express()
 const port = process.env.PORT || 3000
 
+// What is a better file/directory to place this in?
+async function getFestivalData() {
+  // Maybe we need to retreive this data from the database in the future...
+  const musicGenres = [
+    'deephouse',
+    'electro',
+    'hardcore',
+    'hardstyle',
+    'house',
+    'pop',
+    'rb',
+    'rock',
+    'techno',
+    'trance',
+    'urban',
+  ]
+
+  puppeteer.launch().then(async browser => {
+    const page = await browser.newPage()
+    let mergedData = []
+
+    // this has to be an oldschool for loop, because forEach, map and
```

Comment:
and? XD

Reply:
"map, forEach, reduce, etc, does not wait for the asynchronous operation within them."
https://stackoverflow.com/a/46297546

Reply:
You can write it like this?

for (const [i, genre] of musicGenres.entries()) {
   
}